src/RoleObjects/AccessibilityObject.ts

```diff
@@ -107,6 +109,16 @@ export default class AccessibilityObject {
     this.text = undefined;
   }
 
+  // Called to mark the object for updating
+  markForUpdate() {
+    this._markedForUpdate = true;
```

Comment:
If here you also call `this._parent?.markForUpdate()` then you can use  breadth first and use `filter (node) {...}` to ignore branches that don't need visiting/updating.